src/config.ts

```diff
@@ -114,6 +118,8 @@ export const getConfig = (): Config => {
   const imageDirectoryPath = core.getInput('image-directory-path');
   validateGitHubToken(githubToken);
   validateImageDirPath(imageDirectoryPath);
+  const expectedImagesDirectoryPath = core.getInput('expected-images-directory-path') ?? null;
+  validateImageDirPath(imageDirectoryPath, true);
```

Comment:
maybe it is not needed to add `allowEmpty` flag.
```suggestion
if (expectedImagesDirectoryPath) {
  validateImageDirPath(expectedImagesDirectoryPath);
}
```

Reply:
Sorry, bad code from me. Changed accordingly, but using a `null` check instead.